src/main/kotlin/org/jitsi/nlj/transform/node/incoming/Vp8Parser.kt

```diff
@@ -57,6 +59,16 @@ class Vp8Parser(
                 logger.cdebug { "Received a keyframe for ssrc ${videoRtpPacket.ssrc} ${videoRtpPacket.sequenceNumber}" }
                 numKeyframes++
             }
+
```

Comment:
Reading this makes me think we should change the `if (videoRtpPacket is Vp8Packet)` block to:

```
(videoRtpPacket as? Vp8Packet)?.let { vp8Packet ->
    ...
}
```
so we get a more accurate variable name, but no need to do it as part of this, unless you're up for it.